ReactiveDataDisplayManagerTests/REFACTORING/Table/Mock/Generators/MockTableHeaderGenerator.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MockTableHeaderGenerator.swift
+//  ReactiveDataDisplayManagerTests
+//
+//  Created by Anton Eysner on 08.02.2021.
+//  Copyright © 2021 Александр Кравченков. All rights reserved.
+//
+import UIKit
+
+@testable import ReactiveDataDisplayManager
+
+class MockTableHeaderGenerator: TableHeaderGenerator {
```

Comment:
```suggestion
class StubTableHeaderGenerator: TableHeaderGenerator {
```

Ничего не подменяем и не шпионим, поэтому это всего лишь Stub. Даже не Spy.